cloud-agent/service/server/src/main/scala/org/hyperledger/identus/oid4vci/controller/CredentialIssuerController.scala

```diff
@@ -203,9 +203,9 @@ case class CredentialIssuerControllerImpl(
             )
             .mapError(ue => serverError(Some(s"Unexpected error while issuing credential: ${ue.userFacingMessage}")))
         } yield ImmediateCredentialResponse(credential.value)
-      case None => ZIO.fail(badRequestInvalidProof(jwt = "empty", details = "No proof provided"))
-      // case Some(CwtProof(_, _)) => ??? // TODO
-      // case Some(LdpProof(_, _)) => ??? // TODO
+      case None                 => ZIO.fail(badRequestInvalidProof(jwt = "empty", details = "No proof provided"))
+      case Some(CwtProof(_, _)) => FIXME // I have no idea what this is
```

Comment:
It's OID4VCI spec, we support jwt proof only, but cwt and ldp proofs can be used

Reply:
So what should we do with this?

Reply:
`FeatureNotImplemented` looks valid here

Reply:
ok will use that